core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5018,9 +5018,9 @@ public String toString() {
           .build();
   public static final PropertyKey USER_FILE_CREATE_TTL_ACTION =
       enumBuilder(Name.USER_FILE_CREATE_TTL_ACTION, TtlAction.class)
-          .setDefaultValue(TtlAction.DELETE)
+          .setDefaultValue(TtlAction.FREE)
           .setDescription("When file's ttl is expired, the action performs on it. Options: "
-              + "DELETE (default) or FREE")
```

Comment:
Both DELETE and FREE can be accepted by this config item.

Reply:
> Both DELETE and FREE can be accepted by this config item.

Ah, yes.
Thanks @maobaolong for your review.

Updated.
Thanks.